Source/CombatExtended/CombatExtended/StatWorkers/StatWorker_ArmorCoverage.cs

```diff
@@ -24,7 +24,7 @@ public override float GetValueUnfinalized(StatRequest req, bool applyPostProcess
                 totalCoverage += coverage;
             }
 
-            return totalCoverage > 0 ? weightedArmor / totalCoverage : 0;
+            return totalCoverage > 0 ? weightedArmor : 0;
```

Comment:
What's the reason for this change?  This is likely to have ripple effects elsewhere.

Reply:
This caused pawns to believe that they have more armor coverage than they actually did under some circumstances, for example, wearing ONLY an armor vest would result in having 8mm armor coverage.

Reply:
Ah, that makes sense.  If we assume total coverage is 1, since the sum of all body parts coverage *should* be one, then we can drop the division.

That said, the reason for the `totalCoverage > 0` was to avoid a divide-by-zero.  Without the division, `totalCoverage` can go completely away.